child_sync_wp/tools/wp_sync.py

```diff
@@ -94,36 +94,51 @@ def test_xmlrpc(self):
     def upload_children(self, children):
         """ Push children to Wordpress website.
 
-        1 - Create and upload a CSV file containing all children information
+        New :
+        1 - Create dictionary and send to Wordpress through XMLRPC method
+        2 - Image URL (from Cloudinary) is now part of the post insert and
+        not uploaded as file anymore
+
+        Previously :
+        1 - Create and upload a CSV file containing all children
+        information
         2 - Upload all children pictures
         3 - Call XMLRPC method that will update Wordpress
 
         :param children: compassion.child recordset
         :return: result of xmlrpc call to wordpress (true/false)
         """
-        _logger.info("Child Upload on Wordpress started.")
-        csv_file = self._construct_csv(children)
-        _logger.info(".... CSV file constructed : " + csv_file)
-        with self.sftp.cd(self.wp_csv_path):
-            self.sftp.put(csv_file)
-        _logger.info(".... CSV file uploaded.")
-        remove(csv_file)
-
-        pictures_folder = self._build_pictures(children)
-        _logger.info(".... Pictures generated.")
-        with self.sftp.cd(self.wp_pictures_path):
-            for picture_file in listdir(pictures_folder):
-                self.sftp.put(pictures_folder + '/' + picture_file)
-        _logger.info(".... Pictures uploaded.")
-        shutil.rmtree(pictures_folder, ignore_errors=True)
         result = True
-
         try:
-            result = self.xmlrpc_server.child_import.addChildren(
-                self.user, self.pwd)
+            for child in children:
+                childarray = {}
```

Comment:
Convention for naming variables is to put `_` where spaces would be required. So this would be `child_array` or better `child_values`